src/main/webapp/script.js

```diff
@@ -22,4 +36,28 @@ function createIFrame(text) {
   const liElement = document.createElement('iframe');
   liElement.src = text;
   return liElement;
-}
\ No newline at end of file
+}
+
+function displayCalendar() {
```

Comment:
What's the difference between the initial calendar displayed and the calendar that shows up when "Display Calendar" is clicked?

Reply:
The first calendar displayed comes directly from the embedded link provided by Google Calendar's settings (used to confirm that the link works) and the second calendar displayed is set up so that the calendar src is retrieved & an iframe object is built in the script, then sent to the html. Currently the src is still hardcoded, but it has been set up to accept calendar ID's from another servlet.